server/config.ts

```diff
@@ -221,4 +221,8 @@ export default {
       selfServiceURL: 'https://wyiclient.unilink.prisoner.service.justice.gov.uk:1100',
     },
   ],
+  analytics: {
+    // use staging GA4 tag as fallback
+    siteId: get('GA4_SITE_ID', 'G-4VW039LBEF', requiredInProduction),
```

Comment:
I'd call this `ga4SiteId` or something similar just in case we add more analytics types